app/assets/stylesheets/_overrides.scss

```diff
@@ -26,6 +26,18 @@ h1, h2, h3, h4, h5, h6 {
   }
 }
 
+.off-canvas .menu {
```

Comment:
I thought that this has caused a problem with large screen menu where 'Donate' button got pushed down (but I think this was due to menu item padding change), so I added `.off-canvas`, but it could probably be removed.